frontend/client/src/screens/ManageTeams.js

```diff
@@ -33,15 +33,15 @@ const ManageTeamsBase = observer((props) => {
   }
 
   const onAddMemberSuccess = () => {
-    setToastMessage('Member Email Invitation sent')
+    setToastMessage('Success: member email invitation sent')
     setIsSuccess(true)
     setShowAddMemberModal(false)
     confirmationToast.current.show()
   }
 
   const onAddMemberFailure = (e) => {
     Logging.error(e)
-    setToastMessage('Member Email Invitation failed to send')
+    setToastMessage('Member email invitation failed to send')
```

Comment:
For this one, should we give a more descriptive message based on the error? For example, this can occur if you try and add a user that already exists. Perhaps in that case the Toast could say "Member email invitation failed to send, user already exists." or something like that.

Reply:
Agreed.  See my [most recent commit](https://github.com/covidwatchorg/portal/pull/386/commits/da9db9f8c2317c6b09496cf85ddbde95b0796d88) which addresses this case.  If that is the only concern, feel free to approve and merge!